server/etcdmain/help.go

```diff
@@ -103,6 +105,8 @@ Member:
     Read timeout set on each rafthttp connection
   --raft-write-timeout '` + rafthttp.DefaultConnWriteTimeout.String() + `'
     Write timeout set on each rafthttp connection
+  --feature-gates ''
```

Comment:
Noting we will need to raise corresponding pr for etcd website after merging to add user facing docs.

Reply:
ack. This flag name might still change before we finalize the design of the cluster feature gate, so I will add the doc once that has been decided.